### PR TITLE
CP-570 Add authors, documentation, homepage to pubspec

### PR DESCRIPTION
**Pull request by evanweible-wf**
## Changes
Fill out the pubspec.yaml in preparation for publishing to pub.

## Areas of Regression
- n/a

## Testing
- n/a

## Code Review
@trentgrover-wf
@maxwellpeterson-wf

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,11 @@
 name: fluri
 version: 0.0.0
 description: 'Fluri is a fluent URI library built to make URI mutation easy'
+authors:
```
Comment (evanweible-wf) [inline pubspec.yaml]: We can add the client platform team email later once an @workiva.com one has been setup

File: pubspec.yaml [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+  - Evan Weible <evan.weible@workiva.com>
+  - Max Peterson <maxwell.peterson@workiva.com>
+  - Trent Grover <trent.grover@workiva.com>
+documentation:
```
Comment (evanweible-wf) [inline pubspec.yaml]: Including this but leaving it blank will tell pub to generate the docs automatically and host them on pub.

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by evanweible-wf** [comment]
For reference:
```
› pub publish --dry-run
Publishing fluri 0.0.0 to https://pub.dartlang.org:
|-- .gitignore
|-- LICENSE
|-- NOTICE
|-- README.md
|-- lib
|   '-- fluri.dart
|-- pubspec.yaml
|-- test
|   '-- fluri_test.dart
'-- tool
    |-- coverage.sh
    |-- docs.sh
    '-- test.sh

Package has 0 warnings.
```

**Comment by trentgrover-wf** [comment]
+1
@jayudey-wf ready for merge

**Comment by jayudey-wf** [comment]
QA +1, performed get successfully and tests all pass

merging into master

**Comment by jayudey-wf** [comment]
go @rmconsole-wf

**Comment by jayudey-wf** [comment]
@rosie run_merge_script

**Comment by jayudey-wf** [comment]
@rmconsole-wf

**Comment by rmconsole-wf** [comment]
- [x] All commits reviewed
- [x] QA Review
- [x] Security Review Completed if Applicable:  (N/A)

To remain in SOC1 Compliance the following users cannot merge this PR:
* evanweible-wf

		When this pull is merged I will use the following information:
		Version: fluri 1.0.0
		Release Ticket: CP-573

***
*Last updated on Tuesday, April 28 02:58 PM CST*

**Comment by teresarevious-wf** [comment]
@rmconsole-wf

**Pull request merged**